dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -63,36 +101,11 @@ export const ArticlePage = (props: WebProps | AppProps) => {
 	return (
 		<StrictMode>
 			<Global
-				styles={css`
-					:root {
-						/* Light palette is default on all platforms */
-						${paletteDeclarations(format, 'light')}
-						body {
-							color: ${sourcePalette.neutral[7]};
-						}
-						/* Dark palette only for apps and only if switch turned on */
-						${article.config.switches.darkModeInApps &&
-						renderingTarget === 'Apps'
-							? css`
-									@media (prefers-color-scheme: dark) {
-										${paletteDeclarations(format, 'dark')}
-										body {
-											color: ${sourcePalette.neutral[86]};
-										}
-									}
-							  `
-							: ''}
-					}
-					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
-					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
-					*:focus {
-						${focusHalo}
-					}
-					::selection {
-						background: ${sourcePalette.brandAlt[400]};
-						color: ${sourcePalette.neutral[7]};
-					}
-				`}
+				styles={globalStyles(
+					format,
+					article.config.switches.darkModeInApps,
```

Comment:
We could coerce `boolean | undefined` to `boolean` here:

```suggestion
					!!article.config.switches.darkModeInApps,
```

Reply:
All changed :)